package/utils/busybox/Makefile

```diff
@@ -17,7 +17,7 @@ PKG_SOURCE_URL:=https://www.busybox.net/downloads \
 		http://sources.buildroot.net
 PKG_HASH:=d0f940a72f648943c1f2211e0e3117387c31d765137d92bd8284a3fb9752a998
 
-PKG_BUILD_DEPENDS:=BUSYBOX_CONFIG_PAM:libpam
+PKG_BUILD_DEPENDS:=BUSYBOX_CONFIG_PAM:libpam BUSYBOX_CONFIG_SELINUX:libselinux
```

Comment:
Please remove this commit as it's covered in https://github.com/openwrt/openwrt/pull/3303

Reply:
I would like to wait until #3303 is merged. As before, I would like to keep this branch usable until it is merged.

Reply:
Okay.

Reply:
@aparcar, I just removed the busybox commit, as merge request #3303 was merged.

Reply:
Yea thanks for that!